src/tremolo.ts

```diff
@@ -44,19 +44,20 @@ export class Tremolo extends Modifier {
     const scale = gn ? GraceNote.SCALE : 1;
     const category = `tremolo.${gn ? 'grace' : 'default'}`;
 
-    const y_spacing = this.musicFont.lookupMetric(`${category}.spacing`) * stemDirection;
+    const musicFont = Tables.currentMusicFont();
```

Comment:
Use the global font stack everywhere.

Reply:
No change required.

Reply:
Yes, I am not requesting any changes from you (since I wrote this code, hahah).

I am just leaving comments to explain to any reviewers (e.g., 0xfe) why I made these changes.